.circleci/config.yml

```diff
@@ -325,3 +325,15 @@ workflows:
             branches:
               only:
                 - main
+
+  rebuild:
+    triggers:
+      - schedule:
+          # At 00:00 on Tuesday
+          cron: "0 0 * * TUE"
```

Comment:
Let's just do it everyday--it's not that expensive 😄

Reply:
or maybe `MON-FRI`?

Reply:
Changed to:

```
# At 00:00 every day monday-friday
          cron: "0 0 * * MON-FRI"
```

Reply:
Edited, think it needs to be `1-5` instead of `MON-FRI` here for circleci